source/user-manual/wazuh-dashboard/config-file.rst

```diff
@@ -1,18 +1,72 @@
 .. Copyright (C) 2015, Wazuh, Inc.
 
 .. meta::
-  :description: The Wazuh dashboard includes a configuration file where you can define custom values for several options. Learn more about it in this section.
+  :description: The Wazuh dashboard includes a configuration where you can define custom values for several options. Learn more about it in this section.
 
 .. _wazuh_dashboard_config_file:
 
-Configuration file
-==================
+Configuration
+=============
+
+The configuration related to Wazuh plugins can be defined in:
+
+- **Plugins settings**:
```

Comment:
-  **Wazuh dashboard settings**: You can edit the Wazuh dashboard configuration file.

   -  ``/etc/wazuh-dashboard/opensearch_dashboards.yml`` for installations from packages.
   -  ``/usr/share/wazuh-dashboard/config/opensearch_dashoards.yml`` for Docker installations.

-  **Plugins settings**: Users with privileges to manage the Wazuh indexer REST API can manage the plugins settings from the Wazuh dashboard. Wazuh stores the custom values in a saved object.

   -  **Dashboard management** > **App Settings**: To customize several settings.
   -  **Dashboard management** > **Server APIs**: To customize API host entries.

Reply:
Applied the suggestion on: 7211ef37558ea73c7b1dd235ed7dfe7e2e23f6bb